source/our-stacks/ios/index.md

```diff
@@ -24,6 +24,10 @@ We've started with Swift as soon as version 1.0 was released and since then we h
 ##### Objective-C
 Objective-C is a thing of the past for us, but occasionally we do maintain projects or update applications still written in Objective-C.
 
+### Architecture
```

Comment:
Maybe say we do MVC & Viper?

Reply:
You mean, on a separate line? The text below says viper and mvc.